### PR TITLE
Feature auto message from banning

### PR DESCRIPTION
**Pull request by techaffinity**
Banning message has been modified.

### CODE REVIEW

File: app/tests/test_app.py [hunk 1]
```diff
@@ -625,8 +625,8 @@ def test_report_user(self):
             kwargs={'category_slug': 'articles', 'slug': self.post.slug}))
         self.assertContains(
             resp,
-            'This comment has been reported by the community and '
-            'the user has been banned')
+            'This comment has been removed by a moderator and '
+            'the user has been banned from commenting for 3 days')
```
Comment (miltontony) [inline app/tests/test_app.py]: This should be `Your comment has been flagged by a community member. You will not be able to comment again until tomorrow.`

File: project/templates/app/mobi/app/includes/comments_include_no_likes.html [hunk 2]
```diff
@@ -28,10 +28,10 @@
     </span>
 
     {% if comment.is_removed and comment.user.profile.banned %}
-    <p>{% trans "This comment has been reported by the community and the user has been banned" %}<br><br/>
+    <p>{% trans "This comment has been removed by a moderator and the user has been banned from commenting for 3 days." %}<br><br/>
```
Comment (miltontony) [inline project/templates/app/mobi/app/includes/comments_include_no_likes.html]: This was removed by the community so it should be `Your comment has been flagged by a community member. You will not be able to comment again until tomorrow.`

**Comment by techaffinity** [comment]
Testcase has been updated for banned comment.

**Comment by miltontony** [comment]
replaced by #79

**Pull request closed without merging**